Vagrantfile

```diff
@@ -0,0 +1,28 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+  config.vm.box = "debian/testing64"
+
+  # Disable automatic box update checking. If you disable this, then
+  # boxes will only be checked for updates when the user runs
+  # `vagrant box outdated`. This is not recommended.
+  config.vm.box_check_update = false
```

Comment:
No, this is not standard practice. I was using that to just speed up the process of initializing vagrant, but I forgot to remove it :(. I will delete this line in my updated patch.